src/test/java/core/basesyntax/service/impl/FruitTransactionDataConverterServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.InvalidDataFormatException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverterService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionDataConverterServiceTest {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String VALID_FRUIT_DATA = "type,fruit,quantity"
```

Comment:
Please remember some comments from previous task with tests (It is not needed to create all of this object and move it to constants)